### DISCUSSION TITLE
Attaching the toolbox window to the FCP window

### DISCUSSION THREAD

**Discussion by tomwolsky** [latenitefilms/BRAWToolbox#32] [Feature Requests]
Because of the way the Ventura Stage Manager works, it would be really helpful if there was some way to attached the toolbox window to the FCP window. The toolbox likes to separate itself in Stage Manager, which is clunky and pretty pointless it seems for this application.

**Comment by latenitefilms** [comment]
Unfortunately we have no real control over the Workflow Extension window - Final Cut Pro handles all of that, we just pass along the "contents" of the window to Final Cut Pro. I think this will have to be a Final Cut Pro feature request - sorry!

https://www.apple.com/feedback/finalcutpro/
